include/lemlib/movements/movement.hpp

```diff
@@ -20,7 +20,7 @@ class Movement {
          * @brief Update the movement
          *
          * @note This function should be called in a loop
-         * @note Units are in inches and radians. Angle 0 is right, and increases counterclockwise
+         * @note Angle 0 is right, and increases counterclockwise
```

Comment:
```suggestion
         * @note Angle 0 is east, and increases counterclockwise
```
Change `right` to `east`